utils/instructions/Dual/index.ts

```diff
@@ -404,3 +432,227 @@ export async function getWithdrawInstruction({
+      space,
+      'processed'
+    )
+    //Creating checking account on the fly with same mint as base and owner
+    //made to be more safe - instructions don't have access to main treasury
+    const helperTokenAccount = new Keypair()
+    //run as prerequsite instructions payer is connected wallet
+    prerequisiteInstructions.push(
+      SystemProgram.createAccount({
+        fromPubkey: wallet.publicKey,
+        newAccountPubkey: helperTokenAccount.publicKey,
+        lamports: rent,
+        space: space,
+        programId: TOKEN_PROGRAM_ID,
+      }),
+      //initialized account with same mint as base
+      initializeAccount({
+        account: helperTokenAccount.publicKey,
+        mint: baseMint,
+        owner: form.baseTreasury.isSol
+          ? form.baseTreasury.governance.pubkey
+          : form.baseTreasury.extensions.token?.account.owner,
+      })
+    )
+
+    additionalSerializedInstructions.push(
+      //transfer funds from base treasury to the helper checking account
+      serializeInstructionToBase64(
+        Token.createTransferInstruction(
+          TOKEN_PROGRAM_ID,
+          form.baseTreasury.extensions.transferAddress!,
+          helperTokenAccount.publicKey,
+          //owner is sol wallet or governance same as baseTokenAccount
+          form.baseTreasury.extensions!.token!.account.owner,
+          [],
+          form.numTokens
+        )
+      )
+    )
+
+    const quoteTreasuryAccount = await tryGetTokenAccount(
+      connection.current,
+      form.quoteTreasury.pubkey
+    )
+    const quoteMint = quoteTreasuryAccount?.account.mint
+
+    if (!baseMint || !quoteMint) {
+      return {
+        serializedInstruction,
+        isValid: false,
+        governance: form.baseTreasury?.governance,
+        additionalSerializedInstructions: [],
+        chunkSplitByDefault: true,
+        chunkBy: 1,
+      }
+    }
+
+    const soName = `LSO-${form.optionExpirationUnixSeconds}`;
+    const [issueAuthority, _issueAuthorityBump] = await web3.PublicKey.findProgramAddress(
+      [
+        Buffer.from(utils.bytes.utf8.encode('LSO')),
+        toBeBytes(form.optionExpirationUnixSeconds),
+      ],
+      new PublicKey('DiPbvUUJkDhV9jFtQsDFnMEMRJyjW5iS6NMwoySiW8ki')
+    );
+
+    const configInstruction = await so.createConfigInstruction(
+      form.optionExpirationUnixSeconds,
+      form.optionExpirationUnixSeconds,
+      new BN(form.numTokens),
+      new BN(form.lotSize),
+      soName,
+      //use sol wallet as authority
+      form.payer.extensions.transferAddress!,
+      baseMint,
+      //use helper account as base account
+      helperTokenAccount.publicKey,
+      quoteMint,
+      form.quoteTreasury.pubkey,
+      form.payer.extensions.transferAddress!,
+      issueAuthority,
+    )
+
+    additionalSerializedInstructions.push(
+      serializeInstructionToBase64(configInstruction)
+    )
+
+    //after everything we close helper account
```

Comment:
Is this new compared to the update to Staking Options from https://github.com/solana-labs/governance-ui/commit/dafd05dfa77a5145110a6a89579983919db7ba94 ?

Reply:
This is a more complete version of the poorly named Staking Options instruction in realms currently. This is similar except it doesnt do the follow up initStrike and it also uses a separate issue authority

Reply:
Specifically referring to the closure of the helper account. That wasn't done in Staking Options gov UI and had to have a separate proposal for closing old accounts, correct? If so maybe worth adding there too with this PR to simplify it for mango

Reply:
Are you saying the current StakingOptions governance ui instruction isnt working?
As far as I am aware, this still works.
https://github.com/Dual-Finance/governance-ui/blob/133fa4c0af844ecadddb76f04576ee75980765d6/utils/instructions/Dual/index.ts#L226

Reply:
You're right. I had missed this, thought the edits were only to withdraw fct but it's also included in config